packages/wonder-blocks-timing/src/hooks/internal/use-simple-interval.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import {useEffect} from "react";
+
+import {useUpdatingRef} from "./use-updating-ref.js";
+
+/**
+ * A simple hook for using `setInterval`.
+ *
+ * @param action called every `intervalMs` when `isSet` is true
+ * @param intervalMs the duration between calls to `action`
+ * @param isSet whether or not the interval is active
+ */
+export function useSimpleInterval(
+    action: () => mixed,
+    intervalMs: number,
+    isSet: boolean,
+) {
+    const actionRef = useUpdatingRef(action);
+
+    useEffect(() => {
+        if (isSet) {
+            const intervalId = setInterval(() => {
+                actionRef.current();
+            }, intervalMs);
+
+            return () => {
+                clearInterval(intervalId);
+            };
+        }
+        // react-hooks/exhaustive-deps doesn't require refs to be
+        // listed in the deps array.  Unfortunately, in this situation
+        // it doesn't recognized actionRef as a ref.
+    }, [action, actionRef, intervalMs, isSet]);
```

Comment:
Did you mean to remove action from this dep list?

Reply:
It's gone now.

Reply:
Might've forgotten to push up that change, it's still here! 😱